script/unicode/Unicode/Reference.lean

```diff
@@ -0,0 +1,17 @@
+/-
+Copyright (c) 2024 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jean-Baptiste Tristan
+-/
+import Unicode.Unicode
+
+def referenceTable (ucd : List UnicodeData) (property : UnicodeData → Bool) : List Nat :=
+  (ucd.filter property).map (fun ucdc => ucdc.codepoint)
+
+def referenceSearch (table : List Nat) (c : Char) : Bool :=
+  table.contains c.toNat
+
+@[simp]
```

Comment:
No because the search ends up reading the table from a file. Since there are only up to about 1.2 million characters, I wrote the verify program to ensure that the implementation is correct "by reflection".

Reply:
Done